docs/summary/summary.yaml

```diff
@@ -0,0 +1,7 @@
+Benefit expenditure: 222bn
+Employment income: 832bn
+Income Tax revenue: 180bn
+National Insurance (employee-side) revenue: 64.2bn
+Poverty rate (AHC): 18.78%
+Poverty rate (BHC): 15.95%
+Total income: 1tr
```

Comment:
could we add some decimals?

Reply:
Opted for straight-forward rounding to 2dp